tasks/f35/f35_mariadb.yml

```diff
@@ -82,32 +82,33 @@
 # If this decision is ever flipped, the following tasks can be used to create the user with the mysql_user
 # task instead:
```

Comment:
i think it has flipped so we should use it

Reply:
For some reason, in some environments (molecule Ubuntu VM + F35 container), the candlepin user is already created before we get here. To work around this, rather than modifying the root user's permissions, we can change the CREATE USER statement to skip if it already exists:

```
- name: "Create the 'candlepin' MariaDB user"
  become: true
  ansible.builtin.command: "mysql --user root --execute=\"CREATE USER IF NOT EXISTS 'candlepin'@'localhost'; GRANT ALL PRIVILEGES on candlepin.* TO 'candlepin'@'localhost' WITH GRANT OPTION;\""
  changed_when: false
  tags:
    - candlepin
    - mariadb
```

Since we're using this role for more than just dev environment deployments, we should avoid making the more heavy-handed changes to permissions and the like.